lib/Alchemy/Phrasea/WorkerManager/Worker/SubdefCreationWorker.php

```diff
@@ -170,15 +171,26 @@ public function process(array $payload)
                 $this->indexer->flushQueue();
 
                 // tell that we have finished to work on this file
-                $em->beginTransaction();
+                $this->repoWorker->reconnect();
+                $em->getConnection()->beginTransaction();
                 try {
                     $workerRunningJob->setStatus(WorkerRunningJob::FINISHED);
                     $workerRunningJob->setFinished(new \DateTime('now'));
                     $em->persist($workerRunningJob);
                     $em->flush();
                     $em->commit();
                 } catch (\Exception $e) {
-                    $em->rollback();
+                    try {
```

Comment:
i'm not fan of transaction inside transaction. better ? (to be tested to see if the bug is still fixed)
```
for(ntry=0; ntry<10; ntry++) {
  begin_trans
  try {
     do_trick
     commit
     break
  } catch {
     rollbk
     sleep 1
  }
}
if( ntry == 10) {
  shit alors !
}
```